Modules/vis/visualisations/common/vis.helper.js

```diff
@@ -41,38 +41,7 @@ var view =
   {
     this.start = ((new Date()).getTime())-(3600000*24*time);	//Get start time
     this.end = (new Date()).getTime();	//Get end time
-  },
-  
-  'round_interval':function(interval)
```

Comment:
Sorry havent had another chance to look at this, but I think we need to keep track of the effect of removing this. I dont see it as an issue with the engine as its more the way the request is designed to return data closest to request parameters defined by start time and interval. 
Rounding the interval is also nice in that the interval of the data in the graph adhere to values that are more likely to be an integer number of the base data interval.